wiki/Guides/Tablet_Purchase/en.md

```diff
@@ -46,24 +46,39 @@ At the time of writing there are three versions of the Bamboo, all confusingly u
 
 The other Bamboo models come in two sizes, basically small and large. For the second generation, Bamboo Touch, Pen, and Pen and Touch were the "small" ones, whereas the Fun and Craft were the "large" ones. For the third generation, the Connect and Capture are the "small", whereas the Create is the "large".
 
 For osu!, you'll probably be looking at getting one of the "small" sized tablets because it's a common strategy to reduce play area so that you don't have to move your arm when you play. However, it's personal preference as to whether or not you want a larger tablet area, and it's especially worth considering the "large" size if you are also into digital art; drawing is much more comfortable on a larger tablet.
 
-Lastly, Intuos3 tablets can be had for very cheap on second-hand sites like eBay or Craigslist, and are definitely worth checking out if you're also going to be doing digital art as well. For osu!, however, the differences between the Intuos and Bamboo lines are negligible, the only real difference is that the Intuos lines have a much wider range of sizes than is available for Bamboo.
+Lastly, CTL-480 tablets can be had for very cheap on second-hand sites like eBay or Amazon, although they are often hard to find, due to the discontinuation of the manufacturing, but if you could find one for a reasonable price, they are definitely worth checking out if you're also going to be doing digital art as well. For osu!, however, the differences between the Intuos and Bamboo lines are negligible, the only real difference is that the Intuos lines have a much wider range of sizes than is available for Bamboo.
 
-In short, Wacom is the recommended brand from most serious osu! players. Cintiq and Intuos tablets are too expensive and give no real advantage over Bamboo for osu!. See the recommend options above for the recommend models. Also try your luck at a used Intuos3 if you're also into art.
+### Other tablet brands
+
+Other than Wacom and Huion, there are also more tablet manufacturers that tries to compete with the market, bringing a decent product that targetted at artists, and even osu! players.
+
+Some of those manufacturer and their known tablet are:
+
+- Gaomon (known for their Gaomon S620 tablet).
+- XP-Pen (known for their XP-PEN G640 Rev A tablet).
+- VEIKK  (known for their VEIKK S640 tablet).
+
+All of those aforementioned tablets have a pretty decent quality. Often being priced much cheaper than Wacom's tablets, making them a really good tablet when you're on budget.
+
+Although, some people still prefer the Wacom's tablets, despite of those alternatives.
+
+In short, Wacom is the recommended brand from most serious osu! players. Cintiq and Intuos tablets are too expensive and give no real advantage over Bamboo for osu!. See the recommend options above for the recommend models. Also try your luck at a used CTL-480 if you're also into art.
```

Comment:
I think everything else is excessive; besides, there's a section with the recommendations right below
```suggestion
In short, Wacom is recommended by most serious osu! players. Cintiq and Intuos tablets are too expensive and give no real advantage over Bamboo for osu!.
```

Reply:
got it